### DISCUSSION TITLE
Date picker

### DISCUSSION THREAD

**Discussion by Wagfors** [vuetifyjs/vuetify#16415] [General]
Any one know when this will be available in Vuetify 3

**Comment by 1Luc1** [comment]
The (labs) docs state Q2 2023: https://next.vuetifyjs.com/en/labs/introduction/#up-next

The corresponding task is here: https://github.com/vuetifyjs/vuetify/issues/13480
With the info from 2 days ago from @johnleider:
> This task is already in process by @nekosaur https://github.com/vuetifyjs/vuetify/tree/feat/v3-date-picker

**Reply by Wagfors** [reply]
Thanks, Farley new to GitHub hard to navigate :)
